examples/src/main/java/com/datastax/oss/driver/examples/astra/AstraReadCassandraVersion.java

```diff
@@ -13,36 +13,38 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.datastax.oss.driver.examples.apollo;
+package com.datastax.oss.driver.examples.astra;
 
 import com.datastax.oss.driver.api.core.CqlSession;
 import com.datastax.oss.driver.api.core.cql.ResultSet;
 import com.datastax.oss.driver.api.core.cql.Row;
 import java.nio.file.Paths;
 
 /**
- * Connects to a DataStax Apollo cluster and extracts basic information from it.
+ * Connects to a DataStax Astra cluster and extracts basic information from it.
  *
  * <p>Preconditions:
  *
  * <ul>
- *   <li>A DataStax Apollo cluster is running and accessible.
- *   <li>A DataStax Apollo secure connect bundle for the running cluster.
+ *   <li>A DataStax Astra cluster is running and accessible.
+ *   <li>A DataStax Astra secure connect bundle for the running cluster.
  * </ul>
  *
  * <p>Side effects: none.
  *
  * @see <a
- *     href="https://helpdocs.datastax.com/gcp/dscloud/apollo/dscloudGettingStarted.html#dscloudCreateCluster">
- *     Creating an Apollo Database</a>
- * @see <a href="https://helpdocs.datastax.com/gcp/dscloud/apollo/dscloudShareClusterDetails.html">
- *     Providing access to Apollo databases</a>
- * @see <a href="https://helpdocs.datastax.com/gcp/dscloud/apollo/dscloudObtainingCredentials.html">
- *     Obtaining Apollo secure connect bundle</a>
- * @see <a href="https://docs.datastax.com/en/developer/java-driver/4.3">Java driver online
+ *     href="https://docs.datastax.com/en/astra/gcp/doc/dscloud/astra/dscloudGettingStarted.html#dscloudCreateCluster">
```

Comment:
These instructions are for GCP it seems, don't we have more general instructions for any cloud provider?

Reply:
Unfortunately, we don't have generic instructions. We only have AWS and GCP atm.

Reply:
OK, how about we change the link texts to "blahblah (GCP)"?

Reply:
Done